src/twisted/internet/test/test_tcp.py

```diff
@@ -630,9 +632,23 @@ def makeConnection(self, transport):
         clientFactory.protocol = CheckAddress
 
         def connectMe():
-            reactor.connectTCP(
-                fakeDomain, server.getHost().port, clientFactory,
-                bindAddress=(self.interface, port))
+            while True:
+                ignored, port = findFreePort(self.interface, self.family)[:2]
+                bindAddress = (self.interface, port)
+                log.msg("Connect attempt with bindAddress {}".format(bindAddress))
```

Comment:
[`twistedchecker` says this line is too long.](https://travis-ci.org/twisted/twisted/jobs/343852852#L540)